docs/src/content/docs/components/input-select.mdx

```diff
@@ -49,7 +49,13 @@ Displays a dropdown field, allowing a user to select an option from a pre-define
           initialValue: [
             { label: 'Option 1', value: 'option1' }
           ],
-        })
+        }),
+        // since the id of the input above is "selectInput",
+        // access the option's label like so
+        Display.text("Selected label: {{selectInput.label}}"),
+        // since the id of the input above is "selectInput",
+        // access the option's value like so
+        Display.text("Selected value: {{selectInput.value}}")
```

Comment:
Btw. I would suggest using for description this feature from the expressive code library we are using here.

https://expressive-code.com/key-features/text-markers/#adding-labels-to-line-markers

<img width="753" alt="image" src="https://github.com/buttonize/docs/assets/6282843/0b377f63-1582-4df5-9d77-8b7354eb20b5">